static/app/utils/profiling/tableRenderer.tsx

```diff
@@ -20,17 +16,18 @@ export function renderTableHead<K>({
   generateSortLink,
   rightAlignedColumns,
   sortableColumns,
+  onClick,
 }: TableHeadProps<K>) {
   return (column: GridColumnOrder<K>, _columnIndex: number) => {
     return (
       <SortLink
+        onClick={e => onClick?.(column, e)}
         align={rightAlignedColumns?.has(column.key) ? 'right' : 'left'}
         title={column.name}
-        direction={
-          currentSort?.column === column.key ? currentSort?.direction : undefined
-        }
+        direction={currentSort?.key === column.key ? currentSort?.order : undefined}
         canSort={sortableColumns?.has(column.key) || false}
         generateSortLink={generateSortLink?.(column.key) ?? (() => undefined)}
+        replace
```

Comment:
What do you think about handling this from the onClick vs introducing a new prop?

Reply:
i had thought about it and figured it'd be better to align to `react-router`'s more to date api; https://v5.reactrouter.com/web/api/Link/replace-bool  ... we're obviously a couple versions behind and its not currently supported

my other thought around it was that this `SortLink` takes a `generateSortLink`,  with the intent of it being responsible for the actual routing action. by moving the routing to an imperative call inside `onClick` it overrides the utility of `SortLink` and it just becomes a pure UI component.  

i'm good with either way

i found parts of Grid api weird; this is a good example

Reply:
Good idea with the forward compatibility 🙏🏼 (I like the imperative call too, but this is not a big deal either way)